cli/cmd/sync_v1.go

```diff
@@ -153,75 +102,40 @@ func syncConnectionV1(ctx context.Context, cqDir string, sourceSpec specs.Source
 		progressbar.OptionShowCount(),
 		progressbar.OptionClearOnFinish(),
 	)
-	failedWrites := uint64(0)
-	totalResources := uint64(0)
-	for i, destination := range destinationsSpecs {
-		i := i
-		destination := destination
-		g.Go(func() error {
-			var destFailedWrites uint64
-			var err error
-			if err = destClients[i].Write2(gctx, sourceSpec, tables, syncTime, destSubscriptions[i]); err != nil {
-				return fmt.Errorf("failed to write for %s -> %s: %w", sourceSpec.VersionString(), destination.VersionString(), err)
-			}
-			// call Close on destination client using the outer context, so that it happens even if writes get cancelled
-			if err := destClients[i].Close(ctx); err != nil {
-				return fmt.Errorf("failed to close destination client for %s -> %s: %w", sourceSpec.VersionString(), destination.VersionString(), err)
-			}
-			atomic.AddUint64(&failedWrites, destFailedWrites)
-			return nil
-		})
-	}
-
-	g.Go(func() error {
-		t := time.NewTicker(1 * time.Second)
-		defer func() {
-			for i := range destSubscriptions {
-				close(destSubscriptions[i])
-			}
-			t.Stop()
-		}()
-		for {
-			select {
-			case resource, ok := <-resources:
-				if !ok {
-					return nil
-				}
-				totalResources++
-				_ = bar.Add(1)
-				for i := range destSubscriptions {
-					select {
-					case <-gctx.Done():
-						return gctx.Err()
-					case destSubscriptions[i] <- resource:
-					}
-				}
-			case <-t.C:
-				_ = bar.Add(0)
```

Comment:
This change here seems like a regression; previously the ticker ensured that the bar would update at least once a second, so it didn't seem like the CLI was frozen during a long initialization process or period where no resources are loaded (e.g. all the API endpoints return no data)

Reply:
Fixed in https://github.com/cloudquery/cloudquery/pull/10565/commits/8eb5895d1c66d747803a905390d55d806149deed